core/src/main/scala/cats/syntax/contravariantMonoidal.scala

```diff
@@ -25,9 +25,10 @@ package syntax
 import cats.ContravariantMonoidal
 
 trait ContravariantMonoidalSyntax {
-  implicit final def catsSyntaxContravariantMonoidal[F[_], A](
+  @deprecated("Kept for binary compatibility", "2.10.0")
+  final def catsSyntaxContravariantMonoidal[F[_], A](
     fa: F[A]
-  )(implicit F: ContravariantMonoidal[F]): ContravariantMonoidalOps[F, A] =
+  )(F: ContravariantMonoidal[F]): ContravariantMonoidalOps[F, A] =
```

Comment:
Whenever we remove `implicit` I think we should also uncurry the parameter list.